src/Api/Vault/Controllers/CiphersController.cs

```diff
@@ -648,7 +648,7 @@ private async Task<bool> CanEditItemsInCollections(Guid organizationId, IEnumera
         if (await CanEditAllCiphersAsync(organizationId))
         {
             // TODO: This can likely be optimized to only query the requested ciphers and then checking they belong to the org
-            var orgCollections = (await _collectionRepository.GetManyByOrganizationIdAsync(organizationId)).ToDictionary(c => c.Id);
+            var orgCollections = (await _collectionRepository.GetManySharedCollectionsByOrganizationIdAsync(organizationId)).ToDictionary(c => c.Id);
```

Comment:
This is an authorization check - I'm not sure it needs to be limited to shared collections only. You can edit items in your default collection.